msteams-platform/tabs/how-to/using-fluid-msteam.md

```diff
@@ -0,0 +1,322 @@
+---
+title: Using Fluid with Microsoft Teams
+author: timtwang
+ms.author: mobajemu
+description: Tutorial for integrating Fluid-powered real-time collaboration features into a Microsoft Teams tab application
+ms.localizationpriority: medium
+ms.topic: conceptual
+---
+
+# Use Fluid with Microsoft Teams
+
+This tutorial helps to integrate Fluid-powered real-time collaboration features into a [Microsoft Teams tab application](/microsoftteams/platform/tabs/what-are-tabs). By the end of this tutorial you'll be able to integrate any Fluid-powered applications into Microsoft Teams and collaborate with others in real-time.
+
+In this section you'll learn the following concepts:
+
+1. How to integrate a Fluid client into a Microsoft Teams tab application.
+2. How to run and connect your Teams application to a Fluid service (Azure Fluid Relay).
+3. How to create and get Fluid Containers and pass them to a React component.
+
+For an example of how this recipe may be used to build a more complex application, check out the [Teams Fluid Hello World](https://github.com/microsoft/FluidExamples/tree/main/teams-fluid-hello-world) example in our FluidExamples repo.
+
+## Prerequisites
+
+This tutorial assumes that you are familiar with the following concepts and materials
```

Comment:
Let's tweak the verbiage here. . .

This tutorial requires familiarity with the following concepts and resources: